.env.example

```diff
@@ -1,6 +1,6 @@
 CSMM_HOSTNAME=http://localhost:1337
 CSMM_LOGLEVEL=info
-PORT=1337
```

Comment:
I think PORT is sails, while CSMM_PORT is CSMM

CSMM_PORT is only in config/env/production